examples/sound/Sound.cpp

```diff
@@ -93,6 +93,9 @@ int main()
     // Play music from a flac file
     playMusic("ding.flac");
 
+    // Play music from a mp3 file
+    playMusic("ding.mp3");
```

Comment:
We play all the sounds/music at the same time, over each other?

Reply:
They busy-loop inside the playMusic function 😉 

https://github.com/SFML/SFML/blob/master/examples/sound/Sound.cpp#L66-L75

Reply:
Blocking call, played one by one.

Reply:
I see, thanks!